jwst/assign_wcs/tests/test_wfss.py

```diff
@@ -0,0 +1,392 @@
+
+    # refactored call
+    # x, y = wcstools.grid_from_bounding_box(slit.meta.wcs.bounding_box)
+    # wavelength = compute_wavelength_array(x, y, np.zeros(x.shape)  +x0, np.zeros(y.shape)+y0, np.zeros(x.shape)+order)
+    assert_allclose(wavelength, expected)
+
+    with pytest.raises(ValueError):
+        slit = create_slit(model, x0, y0, 3)
+        compute_wavelength_array(slit)
+
+
+def test_NIRCAMBackwardDispersion():
+    forward_ymodels = [Polynomial1D(1, c0=0.5911538461431823, c1=0.000386153846153726),
+                       Polynomial1D(1, c0=-0.1659615384582264, c1=0.0001930769230768787)]
+
+    forward_xmodels = [Polynomial1D(1, c0=0., c1=0.), Polynomial1D(1, c0=0., c1=0.)]
+    forward_lmodels = [Polynomial1D(1, c0=2.4, c1=2.6), Polynomial1D(1, c0=2.4, c1=2.6)]
+
+    forward_model = transforms.NIRCAMForwardColumnGrismDispersion([1, 2], forward_lmodels,
+                                                                  forward_xmodels, forward_ymodels)
+
+
+    ymodels = [Polynomial1D(1, c0=-1530.8764939967652, c1=2589.641434263754),
+               Polynomial1D(1, c0=859.5617529710912, c1=5179.282868527087)]
+
+    xmodels = [Polynomial1D(1, c0=0., c1=0.),
+               Polynomial1D(1, c0=0., c1=0.)]
+    lmodels = [Polynomial1D(1, c0=-0.923076923076923, c1=0.3846153846153846),
+               Polynomial1D(1, c0=-0.923076923076923, c1=0.3846153846153846)]
+    model = transforms.NIRCAMBackwardGrismDispersion([1, 2], lmodels, xmodels, ymodels)
+    wavelength = np.array([[4.724638, 4.724638, 4.724638, 4.724638, 4.724638, 4.724638],
+                           [4.725642, 4.725642, 4.725642, 4.725642, 4.725642, 4.725642],
+                           [4.726646, 4.726646, 4.726646, 4.726646, 4.726646, 4.726646],
+                           [4.72765 , 4.72765 , 4.72765 , 4.72765 , 4.72765 , 4.72765 ],
+                           [4.728654, 4.728654, 4.728654, 4.728654, 4.728654, 4.728654],
+                           [4.729658, 4.729658, 4.729658, 4.729658, 4.729658, 4.729658]])
+
+    x0 = 913.7
+    y0 = 15.5
+    order = 1
+
+    slit = create_slit(forward_model, x0, y0, order)
+    grid = grid_from_bounding_box(slit.meta.wcs.bounding_box)
+    shape = grid[0].shape
+    xdx = np.empty(shape, dtype=np.float64)
+    ydy = np.empty(shape, dtype=np.float64)
+    for j in range(shape[0]):
+        for i in range(shape[1]):
+            xdx[j, i], ydy[j, i], _, _, _ = model(grid[0][j, i], grid[1][j, i],
+                                                  wavelength[j, i], 1)
+
+    expected_xdx = np.array([[20., 21., 22., 23., 24., 25.],
+                             [20., 21., 22., 23., 24., 25.],
+                             [20., 21., 22., 23., 24., 25.],
+                             [20., 21., 22., 23., 24., 25.],
+                             [20., 21., 22., 23., 24., 25.],
+                             [20., 21., 22., 23., 24., 25.]])
+
+    expected_ydy = np.array([[1584.5, 1584.50000003, 1584.5, 1584.5, 1584.5, 1584.5],
```

Comment:
What level of precision does `assert_allclose` use? I'm wondering if you need to specify "1584.50000003" or would "1584.5" be good enough.